README.md

```diff
@@ -448,6 +448,29 @@ scale: 0.001 # floating point factor applied to result, e.g. for Wh to kWh conve
 timeout: 30s # error if no update received in 30 seconds
 ```
 
+### SMA/Speedwire (read only)
+
+The `sma` plugin provides an interface to SMA devices via the Speedwire protocol.
+
+Sample configuration (read only):
+
+```yaml
+source: sma
+uri: 192.168.4.51 # alternative to serial
+serial: 123456 # alternative to uri
+value: ActivePowerPlus # ID of value to read
+password: "0000" # optional (default: 0000)
+interface: "eth0" # optional
```

Comment:
Anführungszeichen weg

Reply:
beim `interface` entfernt
beim `password` hab ich sie gelassen da das ja sonst in yaml ne Zahl wird